common/main/java/com/couchbase/lite/internal/utils/CookieUtils.java

```diff
@@ -0,0 +1,79 @@
+//
```

Comment:
Could this go into CBLCookieStore?

Reply:
I'm not sure what you mean. CBLCookieStore is an interface.

Reply:
Interfaces can have static methods...

Reply:
Cool, I don't know that.